### PR TITLE
Fix a xsite script issue when using oc or kubectl

### PR DESCRIPTION
**Pull request by ryanzhang**
I think the bash part has some problem, it's not working in my terminal( Fedora 31, bshell).
Please check if this is ok.

### CODE REVIEW

**Comment by dmvolod** [comment]
Looks good, but we should check that this works fine on macOS and RHEL also.
@rigazilla what do you think?

File: manual/xsite-minikube/Makefile [hunk 1]
```diff
@@ -126,8 +126,8 @@ create-cache:
 developer-password = $$(kubectl get secret \
     example-infinispan-generated-secret \
     -o jsonpath="{.data.identities\.yaml}" \
-    | base64 -D \
-    | yq -r .credentials[0].password)
+    | base64 -d \
```
Comment (rigazilla) [inline manual/xsite-minikube/Makefile:129]: MacOS uses `-d` for debug. `base64 --decode` it's probably the portable syntax.

File: manual/xsite-minikube/Makefile [hunk 2]
```diff
@@ -126,8 +126,8 @@ create-cache:
...
+    | yq r - credentials[0].password)
```
Comment (rigazilla) [inline manual/xsite-minikube/Makefile:130]: I'm not sure about this, on my FC31 doesn't work. With the fix to the base64 opts the old version is ok.

File: manual/xsite-openshift/Makefile [hunk 3]
```diff
@@ -99,8 +99,8 @@ create-cache:
 developer-password = $$(oc get secret \
    example-infinispan-generated-secret \
    -o jsonpath="{.data.identities\.yaml}" \
-   | base64 -D \
-   | yq -r .credentials[0].password)
+   | base64 -d \
+   | yq r - credentials[0].password)
```
Comment (rigazilla) [inline manual/xsite-openshift/Makefile:103]: see above comments

**Comment by rigazilla** [comment]
@ryanzhang thanks for the PR, please see my review.

**Comment by rigazilla** [comment]
@ryanzhang I've replace `-d` with the longopt version `--decode` and fixed the filters chain. Please have a look if it sounds good to you, thanks!

File: manual/xsite-minikube/Makefile (hunk 1 above)
Comment (ryanzhang) [reply manual/xsite-minikube/Makefile]: Yes, --decode should be the best option here.

File: manual/xsite-minikube/Makefile (hunk 2 above)
Comment (ryanzhang) [reply manual/xsite-minikube/Makefile]: I need to double check if yq -r would work for me. It seemed to me that yq command is like this:
yq read things.yaml a.b.c
yq r - a.b.c (reads from stdin)

my client info:
uname -a
Linux localhost 5.5.7-200.fc31.x86_64 #1 SMP Fri Feb 28 17:18:37 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
 yq --version
yq version 2.4.1

**Comment by ryanzhang** [comment]
@rigazilla It looks good to me. Thanks!
(I am not  sure about the "yq -r " part, it is ok to me if it works in your end.)

File: manual/xsite-minikube/Makefile (hunk 2 above)
Comment (ryanzhang) [reply manual/xsite-minikube/Makefile]: I see what's going on here. 
It turn out that I used a different "yq" tool which behave slightly differently. 
1) https://github.com/mikefarah/yq accept parameter  "yq r -"
2) https://github.com/kislyuk/yq acept parameter "yq -r "

**Comment by rigazilla** [comment]
> @rigazilla It looks good to me. Thanks!
> (I am not sure about the "yq -r " part, it is ok to me if it works in your end.)

yeah seems ok on my side

**Comment by rigazilla** [comment]
merged, thanks @ryanzhang !

**Pull request merged**